CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -67,6 +67,8 @@ namespace Kartaclysm
 		m_fKartCollisionStat(2.0f),
 		m_fOffroadFactorStat(0.5f),
 		m_fOffroadRumbleFactor(0.05f),
+		m_fAIRubberBandingFactorFirst(0.9f),
+		m_fAIRubberBandingFactorLast(1.05f),
```

Comment:
These names are fine, but I feel like something like `m_fAIRubberBandSlowdownFactor` and `m_fAIRubberBandSpeedupFactor` might be a bit more descriptive.  Do what you will.